src/index.html

```diff
@@ -7,8 +7,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header-logo" href="index.html"><img src="./images/logo.svg" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item"><a class="nav-link is-active" href="index.html">Apple</a></li>
+          <li class="nav-item"><a class="nav-link" href="index.html">Samsung</a></li>
```

Comment:
The page shouldn't reload on the link click. Use anchor links instead 
```suggestion
          <li class="nav-item">
            <a class="nav-link" href="#Samsung">Samsung</a>
          </li>
```